confdgnmi/src/confd_gnmi_demo_adapter.py

```diff
@@ -9,18 +16,56 @@ class GnmiDemoServerAdapter(GnmiServerAdapter):
     # map with XPath, value - both strings
     demo_db = {}
     demo_state_db = {}
+    # we use same lock for demo_db and demo_state_db
+    db_lock = threading.Lock()
     num_of_ifs = 10
     _instance: GnmiServerAdapter = None
+    # config map of config elements
+    # "changes" - array of changes contains  (path, val)  tuples or "break"
+    #             or "send" string
+    # "changes_idx" - current index of processing of "changes" array
+    config = {}
 
     capability_list = [
-        dict(name="tailf-common", organization="", version="2020-06-25"),
+        dict(name="tailf-aaa", organization="", version="2018-09-12"),
```

Comment:
this potentially restricts compatibility with specific ConfD versions -> readme seems not to mention any confd range that is "guaranteed" to work with this demo -> maybe this should be added at least as "tested with ConfD ver. X, may work with others"

Reply:
These are just dummy values in Demo implementation (not using ConfD). I use ConfD capabilities so I can reuse tests more easily.

Reply:
no changes